docs/specification.md

```diff
@@ -0,0 +1,169 @@
+## General Specification for the project !   
+> This Specification is only for the MVP of this project that need to do the first user journey , consisting of opening a file editing and saving it into the filesystem !    
+--- 
+> This Specification can't be finished now , since m too retarded to give it soo much time !   
```

Comment:
be a little more professional, its an open source project bro 😓

Reply:
i intentionally dont use gramarly or any intellisense so i can improve my lang ! 😔